Podfile

```diff
@@ -83,7 +83,8 @@ target 'WooCommerce' do
   pod 'Gridicons', '~> 1.2.0'
 
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressAuthenticator', '~> 7.2.0'
+  # pod 'WordPressAuthenticator', '~> 7.2.0'
+  pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'trunk'
```

Comment:
Hello @itsmeichigo 👋 I was checking the latest Authenticator pod version and noticed the latest commit points this to the repo trunk branch now. Do you know if the process has changed from a beta version to the trunk branch? I don't see draft [release notes in the Authenticator repo](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases) anymore, not sure if I missed some process change announcements.

Reply:
I think the latest process was written here: paaHJt-4lp-p2. As soon as you push a new tag in the repo, a job is run to release a new beta version of the library. So you don't need to draft new beta release notes any more.

Reply:
After reading that post again, I think tagging a beta version is not necessary either. You can skip that and just point to `trunk` after merging a fix.